src/Compiler/Microsoft.CodeAnalysis.Razor/src/BindTagHelperDescriptorProvider.cs

```diff
@@ -314,7 +314,10 @@ protected override void Collect(ISymbol symbol, ICollection<TagHelperDescriptor>
                             format: (string)constructorArguments[5].Value);
                     }
 
-                    results.Add(tagHelper);
+                    if (tagHelper is not null)
```

Comment:
Should we nullable-enable this file while we're at it?

Reply:
We should, but I opted to just get the bug fix out first. I'll go ahead and add nullability annotations to this PR, since it would've caught this issue.